src/main/kotlin/com/cognifide/gradle/aem/common/instance/local/QuickstartResolver.kt

```diff
@@ -65,24 +65,32 @@ class QuickstartResolver(private val manager: LocalInstanceManager) {
         aem.prop.file("localInstance.quickstart.sdkDir")?.let { set(it) }
     }
 
-    val sdkWorkDir: File? get() = sdk?.let { sdkDir.get().asFile.resolve(it.nameWithoutExtension) }
+    val sdkWorkDir: File get() = sdkDir.get().asFile
+
+    private val sdkVersionFile: File get() = sdkWorkDir.parentFile.resolve("sdk.md5")
```

Comment:
additional file being a sibling to the unpacked sdk dir is to have an ability to always have same path used in docker-compose volumes. this eliminates a need to have additional variable like `dispatcherDir` which was having aem-sdk path including version. localInstance/sdk/aem-sdk-xxxx/aem.jar.
this sdk.md5 allows me to always have localInstance/sdk/aem.jar